docs/faq.md

```diff
@@ -351,29 +351,30 @@ If you run experiments with tabulated benchmarks using the `SimulatorBackend`, a
 
 ### <a name="remote-tuning"></a> I don’t want to wait, how can I launch the tuning on a remote machine?
 
-You can use the remote launcher to launch an experiment on a remote machine. The
```

Comment:
These lines are quite informative, why are we removing them? i suggest we keep them, and in fact add more content to elaborate differences between various backends.

Reply:
It is more confusing to point out two different ways to do remote launching. The so-called "remote launcher" is still there, but does not receive real maintenance. It is kind of superseded by the stuff in benchmarking/

Reply:
I also agree with Mina's point that the previous version was clearer (for the the question "How can I launch the tuning on a remote machine?"). Could we revert the change on this item? It seems to me that the benchmarking functionalities would be better covered by the next point.

Regarding the remote launcher being replaced by benchmarking/, this is indeed true for the benchmarking use-case. However, for the use-case where someone wants to perform one tuning remotely I believe the remote launcher is still the best way.

Reply:
OK, I revert the change here.